cliquet/resource/__init__.py

```diff
@@ -1028,13 +1028,13 @@ def process_record(self, new, old=None):
 
         return annotated
 
-    def postprocess(self, result, action=ACTIONS.READ):
+    def postprocess(self, result, **kwargs):
```

Comment:
Shouldn't we also catch for *args in case action is not passed as a named argument?